server/api_routes/tagRoutes.js

```diff
@@ -61,16 +61,31 @@ router.get('/:id', (req, res, next) => {
 });
 
 //post new tag
-router.post('/', (req, res, next) => {
-  return Tag.findOrCreate(req.body)
-    .then(tag => {
-      console.log(`Successfully posted new tag`);
-      return res.status(201).json(tag);
-    })
-    .catch(e => {
-      console.error(chalk.red(`Failed to post new tag`), e);
-      next(e);
-    });
+router.post('/', async (req, res, next) => {
```

Comment:
eliot has previously given feedback that we should refrain from putting extensive db logic in the routes. this could all be moved to a class method like Image.tagImages for example